conf/log.conf.d/pfsetvlan.conf.example

```diff
@@ -6,8 +6,15 @@ log4perl.rootLogger = INFO, PFSETVLAN
 #log4perl.category.pf.SNMP = WARN
 
 ### Logging for pfsetvlan
-log4perl.appender.PFSETVLAN                                = Log::Dispatch::Syslog
-log4perl.appender.PFSETVLAN.ident                          = packetfence
-log4perl.appender.PFSETVLAN.facility                       = local5
-log4perl.appender.PFSETVLAN.layout.ConversionPattern       = %X{proc}(%X{tid}) %p: [mac:%X{mac}] %m (%M)%n
-log4perl.appender.PFSETVLAN.layout                         = PatternLayout
+log4perl.appender.PFSETVLAN                              = pf::log::FileAppender
+# Replace /usr/local/pf/logs/packetfence.log by /usr/local/pf/logs/pfsetvlan.log to allow
```

Comment:
Either the comment of the parameter below is wrong

Reply:
I am not sure what you want done it sound like you gave up mid sentence.

Reply:
s/of/or/

And anyway my comment was wrong